manifests/importer.pp

```diff
@@ -20,16 +20,27 @@
   ################### 3. Telemetry dashboard ########################################
   # If using puppet_metrics_dashboard:
 
-    #   class { 'puppet_metrics_dashboard':
-      # add_dashboard_examples => true,
-      # overwrite_dashboards   => false,
-      # configure_telegraf     => true,
-      #enable_telegraf        => true,
-      #master_list            => $infranode,
-      #puppetdb_list          => [$pdb],
-      #postgres_host_list     => [$postgres],
-
-      #}
+  #
+  # this is where lists of master_list , puppetdb_list, postgres_host_list hosts are found using a puppetdb query
+  #make sure each list is same format
+
+  #conditions of whether postgres is present or not
+
+  #in the below class 
+
+    $puppet_severs = rsan::get_puppet_servers()
```

Comment:
Can we expose these as parameters instead of just declaring as variables? That way someone should be able to call the class like so and override the defaults if they need to:

```puppet
class { 'rsan::importer':
  puppet_servers => ['server1.company.com', 'server2.company.com'],
}
```

Reply:
So in this instance, I'd be looking to do something like:

```
class { 'rsan::importer':
  puppet_servers  => $puppet_servers,
  puppetdb_hosts => $puppetdb_hosts,
  postgres_hosts   => $postgres_hosts,
}
```

Is that right?

Reply:
Well that's what the user would do when the use RSAN. In order for you to allow them to do that you'll need to allow the `rsan::importer` class to accept parameters that the user passes i.e.

```puppet
class rsan::importer (
  String $some_param = "hello",
) {
```